example.env

```diff
@@ -3,6 +3,7 @@ KORT_ADMIN_USER="define your username"
 KORT_ADMIN_PASSWORD="define your password"
 MONGO_HOST="mongo"
 RATE_LIMIT="define the number of requests per 10 seconds"
+ENVIRONMENT="set to either production or development"
```

Comment:
I would rather prefer this commented out since this is required only for load testing locally. Also may be change the comment to "Uncomment and set the value to development if you want to disable default security headers for local development"